project/src/test/java/com/google/sps/test/PlaceTest.java

```diff
@@ -43,31 +43,43 @@
   public void create_invalidLowRating_throwsIllegalArgumentException() {
```

Comment:
Suggestion: 
Have a method that returns a valid Builder (with all the values set to the valid values) and then modify just the field which is problematic. 
I believe it'd be more readable.

Reply:
that's a great idea! done